core/fluentasserts/core/objects.d

```diff
@@ -35,6 +35,17 @@ struct ShouldObject(T) {
                                            "a `" ~ T.stringof ~ "` instance"),
                                            file, line);
   }
+
+  auto equal(U)(U instance, const string file = __FILE__, const size_t line = __LINE__) {
+    addMessage(" equal ");
+    addValue("`" ~ U.stringof ~ "`");
+    beginCheck;
+
+    bool isEqual = testData.opEquals(instance);
```

Comment:
`object.d` does nice things:
https://github.com/dlang/druntime/blob/master/src/object.d#L138
(`null` handling; symmetric call)

So, you should not call `opEquals` directly:

    bool isEqual = testData == instance;

Reply:
hmm... i see. I did not know that. Thanks!